public/javascripts/filteredDomains.js

```diff
@@ -1,4760 +0,0 @@
-// This is simply a list of domains that I wanted to prevent the user from signing up with to enhance the trust users place in people that have left reviews, and they get used in the auth route to do that.
```

Comment:
You have this huge array in two places, here and in `public/javascripts/script.js`. I'd probably leave it here and remove it from `scripts.js`. It's huge and needs its own file

Reply:
Okay, will do. If I recall, the script.js is doing client-side validation on my form, and the one is doing the server-side.